app/decorators/comment_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class CommentDecorator < ApplicationDecorator
+  delegate :body, :user_id, :page_id
+
+  # Define presentation-specific methods here. Helpers are accessed through
+  # `helpers` (aka `h`). You can override attributes, for example:
+  #
+  #   def created_at
+  #     helpers.content_tag :span, class: 'time' do
+  #       object.created_at.strftime("%a %m/%d/%y")
+  #     end
+  #   end
```

Comment:
remove pls commented code.  If you decide to get this code back ot better to do it with GIT